ming_sim/beat_orchestration.py

```
@@ -224,14 +233,50 @@ def _identity_snippet(characterization: str) -> str:
     return raw.split("；", 1)[0].strip()
 
 
+def create_llm_beat_generator(llm_config: Any) -> BeatGenerator:
+    """创建真实 scene LLM adapter。
+
+    prompt 只陈列已路由的 in-world 材料，不携带字数、段落、结构或格式约束；失败原样抛出，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)